src/app/course/leader-board-page/leader-board-page.component.scss

```diff
@@ -11,3 +11,8 @@
 .clicked {
     background-color: var(--tui-base-02);
 }
+
+.line {
+    height: 1px;
```

Comment:
Let's use `border` instead of `background` for the line.

Reply:
Ok